### ISSUE TITLE
Cannot query field "connectedObject" on type "WpMenuItem"

### ISSUE THREAD

**Issue by brettkrueger** [gatsbyjs/gatsby-source-wordpress-experimental#46]
The error below is when using a fresh site with the start using `gatsby new krux-gatsby https://github.com/gatsbyjs/gatsby-source-wordpress-experimental/` (also tried directly from Tyler's repo and had the same issue). I'm using WordPress 5.4.2, and PHP 7.4, ensured that the environment variable was set (and also updated manually within the gatsby-config.php), and have both versions of the plugin installed. 

This is a headless WordPress installation at krux.wpengine.com, so not sure if that could be interfering, but I wouldn't think so, since the graphql endpoints on the site respond as expected, eg `https://krux.wpengine.com/graphql?query={pages{edges{node{id}}}}`. I also tested disabling the headless mode and the issue seems to persist after removing my .cache folder in the Gatsby project and restarting the develop server.

For what it's worth, I also tried the suggestion in the error to no avail. Any insight would be greatly appreciated! I didn't see any other issues submitted with this particular error, so I assume that I am doing something incorrectly. 😄 

```
  16:13  error  Cannot query field "connectedObject" on type "WpMenuItem". Did you mean "connectedNode"?  graphql/template-strings

✖ 1 problem (1 error, 0 warnings)


File: src/components/menu.js

failed Re-building development bundle - 0.204s
```

Here is the debug info for my machine as well

```
  System:
    OS: macOS 10.15.5
    CPU: (4) x64 Intel(R) Core(TM) i5-7360U CPU @ 2.30GHz
    Shell: 5.7.1 - /bin/zsh
  Binaries:
    Node: 10.16.0 - ~/.nvm/versions/node/v10.16.0/bin/node
    npm: 6.14.6 - ~/.nvm/versions/node/v10.16.0/bin/npm
  Languages:
    Python: 3.7.4 - /Users/brett.krueger/miniconda3/bin/python
  Browsers:
    Chrome: 83.0.4103.116
    Firefox: 77.0.1
    Safari: 13.1.1
  npmPackages:
    gatsby: 2.22.5 => 2.22.5
    gatsby-image: ^2.4.5 => 2.4.13
    gatsby-plugin-chakra-ui: ^0.1.4 => 0.1.4
    gatsby-plugin-netlify-cache: ^1.2.0 => 1.2.0
    gatsby-plugin-react-svg: ^3.0.0 => 3.0.0
    gatsby-plugin-sharp: ^2.6.8 => 2.6.19
    gatsby-source-filesystem: ^2.3.7 => 2.3.19
    gatsby-source-wordpress-experimental: 0.8.0 => 0.8.0
    gatsby-transformer-sharp: 2.5.3 => 2.5.3
    gatsby-wordpress-experimental-inline-images: ^0.0.3 => 0.0.3
  npmGlobalPackages:
    gatsby-cli: 2.12.60
```

**Comment by brettkrueger** [comment]
Some more info to note:

- This is a multisite install (not live, just testing a bunch of stuff)
- When I first submitted this, I was on the latest version of wp-graphql, but testing [henrikwirth's repo](https://github.com/henrikwirth/gatsby-starter-wordpress-twenty-twenty), it gave me an error about needing to use wp-graphql 9.1, so I tested downgrading that and testing on both gatsby projects, but the error persists.

**Comment by brettkrueger** [comment]
Got this working right away using gatsby-source-wordpress@v3. Since this is still experimental, I'm not entirely sure if this issue is a bug, or just my doing something wrong, so I'll leave this open for now to let the authors decide. Let me know if I can provide any more info about the issue I was having! I haven't deleted the projects using the experimental version.

**Comment by techexpresslive** [comment]
> gatsby-source-wordpress@v3

Will you please share the steps. I am also getting the same error and can't able to build

**Comment by TylerBarnes** [comment]
Hi @brettkrueger , from your description it's hard to tell what it could be. Are you able to share a reproduction repo? There have been some major changes to WPGraphQL recently and `connectedObject` is no longer a field on MenuItem nodes. The new field is `connectedNode`

**Comment by brettkrueger** [comment]
@TylerBarnes revisiting this, I think the issue was actually just a mismatch between my installed WPGraphQL version and the gatsby-source-wordpress-experimental dependency. When I updated to GraphQL 0.10.0, it works as expected. Sorry about that!

@techexpresslive

If the thing I did to fix it doesn't work when you try (ensuring you're on gatsby-source-wordpress-experimental with v0.10.0 of the WP GraphQL plugin [which it has a warning for that I must have missed originally])

```
npm install --save gatsby-source-woocommerce 
```

Should get the latest release version, which is v3, then init a new gatsby site for it and copy files over (correct me if I'm wrong there, Tyler): https://www.gatsbyjs.org/packages/gatsby-source-wordpress/
